path_test.go

```diff
@@ -21,7 +21,7 @@ func (*pathSuite) TestGoPathSet(c *gc.C) {
 
 func (*pathSuite) TestTrimGoPath(c *gc.C) {
 	relativeImport := "github.com/foo/bar/baz.go"
-	filename := path.Join(errors.GoPath(), relativeImport)
+	filename := path.Join(errors.GoPath(), "src", relativeImport)
 	c.Assert(errors.TrimGoPath(filename), gc.Equals, relativeImport)
```

Comment:
I don't think this test will pass on Windows. We generate the filename using path.join and trimGoPath uses os.PathSeparator to make the string to match